src/bacnet/basic/service/h_cov.c

```diff
@@ -277,9 +277,10 @@ static int cov_encode_subscription(
  *          would not fit within the buffer.
  */
 /* Maximume length for an encoded COV subscription  - 27 bytes for BACNET IP6
- * 31 bytes for IPv4 (longest MAC) and lets round it up to the machine word
+ * 31 bytes for IPv4 (longest MAC) plus 3 extra bytes for the maximum length
+ * of PID (5 bytes) and lets round it up to the 64bit machine word
```

Comment:
Just on the commit, why are you adding 3 extra bytes if the max length of PID is 5 bytes? It doesn't make sense to me. Also, would the next byte alignment not be 34 (31+3) or 36 (31+5), not 48?

Reply:
48 - thinking 64 bit architecture - possibly 40 would be ok
31 bytes is for pid of min length (2bytes) whereas the max length is 5 bytes hence 3 extra bytes

Reply:
Ameliorated the code comment ;)